index.js

```diff
@@ -1,35 +1,41 @@
 import { createServer } from "http";
-import { Octokit } from "@octokit/rest";
 import nunjucks from "nunjucks";
+import { OctokitApp } from "./octokitApp.js";
 
 nunjucks.configure({
   autoescape: true,
   watch: true,
 });
 
-const ACCESS_TOKEN = process.env.GITHUB_PERSONAL_ACCESS_TOKEN;
+const httpServer = createServer(async (request, response) => {
+  if (await OctokitApp.middleware(request, response)) return;
 
-const octokit = new Octokit({
-  auth: ACCESS_TOKEN,
-});
+  const installations = [];
+  await OctokitApp.app.eachInstallation(async octokit => {
+    const name = octokit.installation.account.login;
+    
+    const repos = await getReposForInstallation(octokit);
 
-const httpServer = createServer(async (_, response) => {
-  const repos = await getRepos({ org: "dxw" });
+    installations.push({
+      name,
+      repos,
+    });
+  });
+
+  installations.sort((a, b) => a.name.localeCompare(b.name));
 
   const template = nunjucks.render("index.njk", {
-    repos,
+    installations,
```

Comment:
I'm not sure that `installation` is helpful terminology for the UI to deal with. 
If I understand the code correctly then I think `organisation` would be more intelligible.

Reply:
"Installations" is the terminology that GitHub uses, as it covers both individual and organisation accounts. In practice I don't think we'd be expecting any individual users to install the app, but if there's a strong preference for `organisation` as a variable name then I'd want to introduce a check so that we ignore installations for individual accounts.

Reply:
I think we should only install the app in the dxw org so I think repos would be clearer

Reply:
I haven't been able to find a way to get a specific installation ID from an account name, so I think we're stuck using `eachInstallation` to iterate.

The best we could probably do is something like:

```js
  const template = nunjucks.render("index.njk", installations[0]);
```

Which would expose `name` and `repos` to the Nunjucks template.